src/components/Garden/ModalFlows/ChallengeProposalScreens/ChallengeRequirements.js

```diff
@@ -6,6 +6,7 @@ import {
   LoadingRing,
   textStyle,
   useTheme,
+  Link,
```

Comment:
We generally alphabetize imports form the same library/file

Could we place it right after `Info`?